_po/ja/how-to/laravel.po

```diff
@@ -143,28 +191,40 @@ msgid ""
 "* `app/Http/Controller/PostController.php` - The HTTP controller that will tak"
 "e incoming requests and return responses."
 msgstr ""
+"* `app/Models/Post.php` - Eloquentのモデルです。\n"
+"* `database/migrations/<timestamp>_create_posts_table.php` - データベースの"
+"テーブルを生成するマイグレーションファイルです。\n"
+"* `app/Http/Controller/PostController.php` - HTTPのリクエストやレスポンスを"
+"取り扱うコントローラです。"
 
 msgid ""
 "#### Model\n"
 "Let’s add search function to our Post model `app/Models/Post.php`."
 msgstr ""
+"#### モデル\n"
+"Postモデルの`app/Models/Post.php`に検索機能を作りましょう。"
 
 msgid "Before part is original state, and After is modified version:"
-msgstr ""
+msgstr "元ファイルと変更後のファイルの変更前、変更後です。"
 
 msgid ""
 "Before:\n"
 "```php\n"
 "<?php"
 msgstr ""
+"変更前:\n"
+"```php\n"
+"<?php"
 
 msgid "namespace App\\Models;"
-msgstr ""
+msgstr "namespace App\\Models;"
 
 msgid ""
 "use Illuminate\\Database\\Eloquent\\Factories\\HasFactory;\n"
 "use Illuminate\\Database\\Eloquent\\Model;"
 msgstr ""
+"use Illuminate\\Database\\Eloquent\\Factories\\HasFactory;\n"
+"use Illuminate\\Database\\Eloquent\\Model;"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.